src/test/java/core/basesyntax/db/StorageTest.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.db;
+
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+class StorageTest {
+    private Storage storage;
+
+    @AfterEach
+    void tearDown() {
+        Storage.fruitShopStorage.clear();
+    }
+
+    @Test
+    void storageKeyAfterPut_Ok() {
+        Storage.fruitShopStorage.put("apple", 10);
```

Comment:
Do you test here that method put works correctly in Java Map ? 
Remove StorageTest

Reply:
I removed  class StorageTest in last commit
https://github.com/mate-academy/jv-fruit-shop-tests/pull/1062/commits/42abf702b40a7fafead1445fb40c58d4552b8e80